.gitignore

```diff
@@ -4,6 +4,10 @@ dist/
 # Dependencies / NPM packages
 node_modules/
 
+# Environment variables (local configuration)
+.env
+.env.local
```

Comment:
```suggestion
```

Is `.env.local` necessary? I don't think we reference it anywhere in the docs or a script. `.env` seems to be the only one used so far.